File: PioneerApp Fadhel/src/gestiontransport/Entite/Station.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package gestiontransport.Entite;

import javafx.beans.property.SimpleDoubleProperty;
import javafx.beans.property.SimpleFloatProperty;
import javafx.beans.property.SimpleIntegerProperty;
import javafx.beans.property.SimpleStringProperty;

/**
 *
 * @author ASUS-PC
 */
public class Station {
    
    private SimpleIntegerProperty id ;
    private SimpleStringProperty Name; 
    private SimpleDoubleProperty Altitude;
    private SimpleDoubleProperty Longtitude;
    private SimpleIntegerProperty Idtrajet; 

    public Station(Integer id, String Name,Double Altitude, Double Longtitude, Integer Idtrajet) {
        this.id = new SimpleIntegerProperty(id);
        this.Name = new SimpleStringProperty(Name);
        this.Altitude = new SimpleDoubleProperty(Altitude);
        this.Longtitude = new SimpleDoubleProperty(Longtitude);
        this.Idtrajet = new SimpleIntegerProperty(Idtrajet);
    }
    
    

    public Integer getId() {
        return id.get();
    }

    public void setId(SimpleIntegerProperty id) {
        this.id = id;
    }

    public void setName(SimpleStringProperty Name) {
        this.Name = Name;
    }
    public String getName() {
        return Name.get();
    }

 

    public Double getAltitude() {
        return Altitude.get();
    }

    public void setAltitude(SimpleDoubleProperty Altitude) {
        this.Altitude = Altitude;
    }

    public Double getLongtitude() {
        return Longtitude.get();
    }

    public void setLongtitude(SimpleDoubleProperty Longtitude) {
        this.Longtitude = Longtitude;
    }

    public Integer getIdtrajet() {
        return Idtrajet.get();
    }

    public void setIdtrajet(SimpleIntegerProperty Idtrajet) {
        this.Idtrajet = Idtrajet;
    }
   
   
}

File: PioneersWeb/src/TransportBundle/Entity/Station.php
<?php

namespace TransportBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Station
 *
 * @ORM\Table(name="station", uniqueConstraints={@ORM\UniqueConstraint(name="Name", columns={"Name"})})
 * @ORM\Entity(repositoryClass="TransportBundle\Repository\StationRepository")
 */
class Station
{
    /**
     * @return int
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * @param int $id
     */
    public function setId($id)
    {
        $this->id = $id;
    }

    /**
     * @return string
     */
    public function getName()
    {
        return $this->name;
    }

    /**
     * @param string $name
     */
    public function setName($name)
    {
        $this->name = $name;
    }

    /**
     * @return int
     */
    public function getIdtrajet()
    {
        return $this->idtrajet;
    }

    /**
     * @param int $idtrajet
     */
    public function setIdtrajet($idtrajet)
    {
        $this->idtrajet = $idtrajet;
    }

    /**
     * @return float
     */
    public function getLongtitude()
    {
        return $this->longtitude;
    }

    /**
     * @param float $longtitude
     */
    public function setLongtitude($longtitude)
    {
        $this->longtitude = $longtitude;
    }

    /**
     * @return float
     */
    public function getAltitude()
    {
        return $this->altitude;
    }

    /**
     * @param float $altitude
     */
    public function setAltitude($altitude)
    {
        $this->altitude = $altitude;
    }
    /**
     * @var integer
     *
     * @ORM\Column(name="id", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $id;

    /**
     * @var string
     *
     * @ORM\Column(name="Name", type="string", length=20, nullable=false)
     */
    private $name;

    /**
     * @var integer
     *
     * @ORM\Column(name="idtrajet", type="integer", nullable=false)
     */
    private $idtrajet;

    /**
     * @var float
     *
     * @ORM\Column(name="longtitude", type="float", precision=10, scale=0, nullable=false)
     */
    private $longtitude;

    /**
     * @var float
     *
     * @ORM\Column(name="altitude", type="float", precision=10, scale=0, nullable=false)
     */
    private $altitude;


}

